### ISSUE TITLE
Request to make query results in drawer a) deletable and b) discernible

### ISSUE THREAD

**Issue by SebastianBehrens** [kristijanhusak/vim-dadbod-ui#178]
First off thank you for this plugin! Fantastic stuff.

![image](https://github.com/kristijanhusak/vim-dadbod-ui/assets/51058351/1637777b-227b-4710-b4df-c3d3a6c2d9f2)

I think it would be nice to either mark results that contain errors differently from properly executed queries without errors.

And also it would be nice if the [deletion key in the drawer](https://github.com/kristijanhusak/vim-dadbod-ui/blob/200aebf70e6fc4b0eb1d3ecf0d57028e198d8dc6/doc/dadbod-ui.txt#L212) could query results.

What do you think?

**Comment by jaycle** [comment]
I agree. Just started playing with dadbod and this was the first thing I tried to figure out. How can we clear these?

**Comment by SebastianBehrens** [comment]
I tried to understand the source code and figured the following; hopefully that helps.
For making buffers deletable: one could add a function that enables deletion of entries in `dbui.dbout_list` ([link](https://github.com/kristijanhusak/vim-dadbod-ui/blob/200aebf70e6fc4b0eb1d3ecf0d57028e198d8dc6/autoload/db_ui.vim#L205C3-L205C40))
For making buffers discernible one could a), add a rename of above mentioned key and/or b) add a function that matches some regex to the value (variable `content`) of the keys in the above mentioned object, and if match exists automatically renames the key.
